tests/test_parser.py

```diff
@@ -326,7 +326,7 @@ def test_new_relation_with_annotations_unidir():
     assert rel.left[2] is None
     assert rel.right[2] == (1, 1)
     assert len(rel.annotations) == 2
-    range1: Range = Range("test", 2, 18, 2, 21)
+    range1: Range = Range("test", 2, 14, 2, 17)
```

Comment:
I assume this was a bug in the equals function?

Reply:
yes, Location had a  __eq__  function that was used by Range as it was not overwritten before.